src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -61,6 +61,23 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceBefore&quot;/&gt;
  * </pre>
+ * <p>Example of violation:</p>
+ * <pre>
+ * int a = a ++ + a --; // '++' and '--' are preceded with whitespace
+ * int foo
+ * ; // allowLineBreaks is set false by default
+ * public int b ;
+ * for (int i = 0 ; i &lt; 5; i++) {}
```

Comment:
what this line is for? if here is violation, there should be a comment that clearly show it